src/modules/imageresizer/ui/Properties/Resources.resx

```diff
@@ -140,13 +140,13 @@
     <value>Custom</value>
   </data>
   <data name="Input_IgnoreOrientation" xml:space="preserve">
-    <value>Ign_ore the orientation of pictures</value>
+    <value>Ignore the _orientation of pictures</value>
   </data>
   <data name="Input_GifWarning" xml:space="preserve">
     <value>Gif files with animations may not be correctly resized.</value>
   </data>
   <data name="Input_Replace" xml:space="preserve">
-    <value>R_esize the original pictures (don't create copies)</value>
+    <value>Over_write files</value>
```

Comment:
Why are we changing from an _e to the _w?

Reply:
No specific reason.